apps/framework-docs-v2/src/config/README.md

```
@@ -0,0 +1,95 @@
+  TypeScript-specific instructions here
+</ConditionalContent>
+
+<ConditionalContent when={{ language: "python" }}>
+  Python-specific instructions here
+</ConditionalContent>
+```
+
+## Hiding/Showing Settings
+
+To temporarily hide a setting without removing it:
+```typescript
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)